release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -1675,6 +1684,8 @@ With this update, the `assisted-installer-controller` can remove the uninitializ
 
 * Previously, the `manila-csi-driver-controller-metrics` service had empty endpoints due to an incorrect name for the app selector. With this release the app selector name is changed to `openstack-manila-csi` and the issue is fixed. (link:https://issues.redhat.com/browse/OCPBUGS-9331[*OCPBUGS-9331*])
 
+* Previously, the assisted installer removed the uninitialized taints for all vSphere nodes which prevented the vSphere CCM from initializing the nodes properly. This caused the vSphere CSI operator to degrade during the initial cluster installation because the node's provider ID was missing. With this release, the assisted installer checks if vSphere credentials were provided in the `install-config.yaml`. If credentials are provided, the OpenShift version is greater or equal to 4.15, and the agent installer was used, the assisted-installer and assisted-installer-controller do not remove the uninitialized taints. This means that the node's providerID and VM's UUID are properly set and the vSphere CSI operator is installed. (link:https://issues.redhat.com/browse/OCPBUGS-29485[*OCPBUGS-29485*])
```

Comment:
```suggestion
* Previously, the assisted installer removed the uninitialized taints for all vSphere nodes which prevented the vSphere CCM from initializing the nodes properly. This caused the vSphere CSI operator to degrade during the initial cluster installation because the node's provider ID was missing. With this release, the assisted installer checks if vSphere credentials were provided in the `install-config.yaml`. If credentials were provided, the OpenShift version is greater or equal to 4.15, and the agent installer was used, the assisted-installer and assisted-installer-controller do not remove the uninitialized taints. This means that the node's providerID and VM's UUID are properly set and the vSphere CSI operator is installed. (link:https://issues.redhat.com/browse/OCPBUGS-29485[*OCPBUGS-29485*])
```